apps/server-asset-sg/src/features/users/user.repo.ts

```diff
@@ -66,7 +66,7 @@ export class UserRepo implements Repo<User, UserId, UserData & { oidcId: string
           isAdmin: data.isAdmin,
           workgroups: {
             deleteMany: {
-              workgroupId: { notIn: data.workgroups?.map((workgroup) => workgroup.workgroupId) },
```

Comment:
I got a little confused by this as well, but the reason i changed this was because the original code would not allow us to update the role of an exisiting workgroup, as it did not get deleted and then skipped in the next step. 
So if we delete all the workgroups related to a user and then add them all again, we can update as well as add new ones